src/resources/vue/pages/terminal/index.vue

```diff
@@ -0,0 +1,79 @@
+<template>
```

Comment:
Ik weet niet in waar je de height hebt aangepast maar de indexpage is nu fricked